dependencies.props

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+1.0.1-prerelease-01401-04<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I guess I still don't understand where this would come from. I tried updating just the CurrentRef's locally and ran UpdateDependencies and it didn't generate this.